widget_driver/README.md

```diff
@@ -372,6 +411,67 @@ Easy...
     }
    ```
 
+### Caveat's when working with dependencies
+
+The driver is bound to the lifecycle of the widget, this means it lifes as long as the widget does. However we do not want to recreate the driver on every state change, that would increase the build time of a `DrivableWidget`. (That's also why we need the mixin for `@driverProvidableProperties` annotated properties)
```

Comment:
This is not really true.. The widget gets created often.. So the widget lifecycle is very short..
Whereas the drivers lifecycle might be longer..
The drivers lifecycle is actually tied to the states lifecycle.. (the state of the statefulWidget).

Maybe also change the title of this section.. Maybe mention something about dependencies which change..
Like model data being passed in... As you mentioned earler today: 
If you pass in a service as the dependnecy.. most of those services are singletons and never change..
I mean their internal data changes.. but the object instance doesnt..
Where as model data might get new instances often.